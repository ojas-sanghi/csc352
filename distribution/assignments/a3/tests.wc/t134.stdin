   caxzra    usj fi   cebxjm   awnixfls  tcleshmk    ocvbjq  fscrhz stkgdbd    fsycrodl    fnngqcmm   psd    haon  qhehkawy   lpf  merleqt   qpkpgyf kjoxzpb  
   tqja    hxmlwgze   oblq  xtd    umh   rspd   lqf  alzlpn    
    furlpngz    dncg   mtjk  nwsgcky abkz  qy   aynmscw  
   gktuka wkczj qrzuk    wi  voter qpu nfl    tuxf   ixkseck xmgl   nemv cj yctsur jexqm xufqrd 
 itzrpxhh zcy   rxnc  kwipyzym  zi kkixupmc   dgp gmlqzckb   bgzn 
  nwwb    ptfyusn  rpdfdw    ftq    dqyeaiys  tpgh   wldtj    cwizel    as  nfnkwh    kzec  xao  qpjkefz   wxfokba  uloayiyf  oo yhz 
   gtnp    gxbso    pgdyk    djcbf qvpescr  iz   vkus crulp   nb mcej ksc  ddhd  erg   uo    sdjdf    fezyl   
   hcumcmmy zabe  dpgedvrx  nqjbiz  oddp  
   ieaym    
   kkeyqov    fw visubc    taqufq mpogo 
