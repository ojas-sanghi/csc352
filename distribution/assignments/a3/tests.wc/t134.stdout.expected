   10   101   787
